### DISCUSSION TITLE
Scheduled recording to Podcast Azuracast

### DISCUSSION THREAD

**Discussion by danielrodrigohub** [AzuraCast/AzuraCast#4257] [Ideas]
I found amazing the podcast section into the azuracast web panel, but would be great that the software automatically records shows from a specific time or day without uploading from the computer. Is that possible?... Many thanks!!!!

**Comment by BusterNeece** [comment]
@danielrodrigotest We absolutely plan on including tighter integration between the Podcasts section and the rest of AzuraCast, and the idea of tying DJ recordings to automatically create podcast episodes is definitely something we'd entertain doing. 

You should submit that as a suggestion over at https://features.azuracast.com/
